python/lsst/sims/utils/fileMaps.py

```diff
@@ -0,0 +1,81 @@
+import re
+
```

Comment:
I don't actually know how this fileMaps is used. Would it make sense at all to have it be part of sims_seds_library? Or not, since that is mostly a data package not a python package?